components/ListElement/src/index.scss

```diff
@@ -0,0 +1,17 @@
+.denhaag-list-element {
```

Comment:
Same here, the CSS can be shorter when `-element` isn't added. In my opinion it's unnecessary.

Reply:
'list' already exists